### ISSUE TITLE
Program hangs with many mixed addresses.

### ISSUE THREAD

**Issue by andrewtwin** [andrewtwin/ip-deaggregator#7]
Program seems to hang indefinably when passed a very long line of subsets to exclude

To reproduce, run with:
```
-n 0.0.0.0/0 10.0.0.1 10.1.0.1 10.128.0.1 10.196.0.1 127.0.0.0/8 172.16.0.0/12 192.168.100.0/24 192.168.196.0/25 224.0.0.0/8 10.0.0.0/8 255.0.0.0/8
```

**Comment by andrewtwin** [comment]
It will eventually complete, but due to the presence of host addresses (with a prefix length of 32) the program considers many more addresses.

**Comment by andrewtwin** [comment]
Removing redundant subsets from the input and sorting the subnets from largest to smallest significantly reduces the run time. 115 minutes -> 24 seconds .

**Comment by andrewtwin** [comment]
Fixed in v1.5.0
